composer.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "joomla/entities",
+  "type": "project",
+  "description": "Joomla Entity Project",
+  "keywords": ["joomla", "entity"],
+  "homepage": "https://github.com/joomla-projects/entities",
+  "license": "GPL-2.0-or-later",
+  "require": {
+    "php": "^7.1.3",
```

Comment:
For now 7.0 please as per the rest of framework 2.0 (e.g. https://github.com/joomla-framework/database/blob/2.0-dev/composer.json#L9)